app.js

```diff
@@ -0,0 +1,135 @@
+  var td = document.createElement('td');
+  td.setAttribute('class', 'location-td');
+  td.textContent = this.location;
+  row.appendChild(td);
+  for(var i = 0; i < this.cookies.length; i++) {
+    td = this.getTableData(i);
+    row.appendChild(td);
+  }
+  td.textContent = this.getTotalCookies();
+  row.appendChild(td);
+  return row;
+};
+
+CookieStand.prototype.getTableData = function(index) {
+  var td = document.createElement('td');
+  td.setAttribute('class', 'sales-td');
+  td.textContent = this.cookies[index];
+  return td;
+};
+
+function createTableHeaderRow() {
+  var row = document.createElement('tr');
+  row.setAttribute('id', 'sales-header-row');
+  var th = document.createElement('th');
+  th.setAttribute('class', 'sales-th');
+  th.textContent = '';
+  row.appendChild(th);
+  var time;
+  for(var i = 1; i < 15; i++) {
+    th = document.createElement('th');
+    th.setAttribute('class', 'sales-th');
+    time = i + 5;
+    if(time < 12){
+      th.textContent = time + ':00am';
+    } else if(time > 12){
+      time -= 12;
+      th.textContent = time + ':00pm';
+    } else {
+      th.textContent = time + ':00pm';
+    }
+    row.appendChild(th);
+  }
+  th = document.createElement('th');
+  th.setAttribute('class', 'sales-th');
+  th.textContent = 'Daily Location Total';
+  row.appendChild(th);
+  return row;
+}
+
+function createTableFooterRow(pike, seaTac, seattleCenter, capitolHill, alki) {
+  var row = document.createElement('tr');
+  row.setAttribute('id', 'sales-footer-row');
+  var tf = document.createElement('td');
+  tf.setAttribute('class', 'sales-tf');
+  tf.textContent = 'totals';
+  row.appendChild(tf);
+  for(var i = 0; i < 14; i++){
+    var total;
+    tf = document.createElement('td');
+    tf.setAttribute('class', 'sales-tf');
+    total = pike.cookies[i] + seaTac.cookies[i] + seattleCenter.cookies[i] + capitolHill.cookies[i] + alki.cookies[i];
+    tf.textContent = total;
+    row.appendChild(tf);
+  }
+  tf = document.createElement('td');
+  tf.setAttribute('class', 'sales-tf');
+  tf.textContent = '';
+  row.appendChild(tf);
+  return row;
+}
+
+function renderTable() {
+  var pike = new CookieStand('1st and Pike', 23, 65, 6.5);
+  pike.getCookiesSold();
+
+  var seaTac = new CookieStand('SeaTac Airport', 3, 24, 1.2);
+  seaTac.getCookiesSold();
+
+  var seattleCenter = new CookieStand('Seattle Center', 11, 38, 3.7);
+  seattleCenter.getCookiesSold();
+
+  var capitolHill = new CookieStand('Capitol Hill', 20, 38, 2.3);
+  capitolHill.getCookiesSold();
+
+  var alki = new CookieStand('Alki', 2, 16, 4.6);
+  alki.getCookiesSold();
+
+  var div = document.getElementById('table-div');
+  var table = document.createElement('table');
+  table.setAttribute('id', 'sales-table');
+  var head = document.createElement('thead');
+  head.setAttribute('id', 'sales-head');
+  head.appendChild(createTableHeaderRow());
+  table.appendChild(head);
+  table.appendChild(pike.getTableRow());
```

Comment:
Much of this appending can be done at the end of the method instead of invoking the method inside of an appending action. It makes it a little cleaner and easier to read.